README.md

```diff
@@ -271,6 +278,98 @@ const result = toTitleCase(str);
 //    ^ 'Hello World'
 ```
 
+## Strongly-typed transformation of objects
+
+### delimiterKeys
+This function recursively converts the keys of an object to a new case with a custom delimiter at both runtime and type levels.
```

Comment:
```suggestion
This function shallowly converts the keys of an object to a new case with a custom delimiter at both runtime and type levels.
```

Reply:
done in [6cde47a](https://github.com/gustavoguichard/string-ts/pull/16/commits/6cde47a1a68c50d506dc516c25ad05ce56454ed6)